bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/ChannelState.java

```diff
@@ -197,23 +199,25 @@ public void processMessage(String topic, byte[] payload) {
             return;
         }
 
-        Command postOnlyCommand = cachedValue.isPostOnly(command);
-        if (postOnlyCommand != null) {
-            channelStateUpdateListener.postChannelCommand(channelUID, postOnlyCommand);
-            receivedOrTimeout();
-            return;
-        }
-
+        Type parsedCommand;
         // Map the string to a command, update the cached value and post the command to the framework
         try {
-            cachedValue.update(command);
+            parsedCommand = cachedValue.parseMessage(command);
```

Comment:
The variable name does not match any more. 
It is now a parsedMessage, right?

Reply:
Fixed with latest commit